apps/api/src/routes/CriteriaRoutes.ts

```diff
@@ -1,19 +1,37 @@
 import { Router } from 'express';
-import { criteriaQuery } from '../services';
+import { database } from '../loaders';
+import { Criteria } from '../models/criteria/CriteriaEntity';
+import { CriteriaService, ICriteriaSearch } from '../services';
 
 const criteriaRouter = Router();
 
 criteriaRouter.get('/', async (req, res) => {
-    const criterias = await criteriaQuery.selectAllCriteria();
+    try {
+        // Hacky, consider assigning string | string[] to queries through request and
+        // split into object with where and sort fields.
+        const queries = req.query as unknown as ICriteriaSearch;
```

Comment:
Is not possible to say: const queries = req.query as ICriteriaSearch | uknown ?

Reply:
I dont know how stringing types in that way works in TS, but i does not seem very error prone :P

Reply:
That does work, but causes TS to interpret `queries` as unknown regardless of its type.

Reply:
Hmm ok, add a TODO to the comment and maby an issue so that we are aware of it in the future